server.go

```diff
@@ -4582,7 +4582,7 @@ func (s *server) SendCustomMessage(peerPub [33]byte, msgType lnwire.MessageType,
 
 	msg, err := lnwire.NewCustom(msgType, data)
 	if err != nil {
-		return err
+		msg, err = lnwire.NewUnknown(msgType, data)
```

Comment:
This may be superfluous. It enables to run:

`sendcustom --peer {alice_pk} --type 32000 --data 437573746f6d206d657373616765207061796c6f61640a`

i.e it enables `sendcustom` to send a message in the 'unknown' range and trigger the connection. However it was just for testing. I included this for the discussion.

Reply:
Yeah i was pretty sure this wasn't going to make it. It was actually for testing. Not sure why i included it.